src/store/time/attendance/sagas.ts

```diff
@@ -132,14 +135,12 @@ function* fetchLeaveRecords(action: FetchLeaveRecordsAction) {
     const response: ApiResponse<LeaveObject[]> = yield apiCall(
       apiGetCall,
       prepare(
-        API_ENDPOINT_LEAVE,
+        API_ENDPOINT_GRAPH_LEAVE_RECORDS,
         {},
         {
           fromDate: action.payload.fromDate,
           toDate: action.payload.toDate,
-          pendingApproval: true,
-          scheduled: true,
-          taken: true,
+          statuses: ['1', '2', '3'],
```

Comment:
```suggestion
          statuses: ['1', '2', '3'], // pending approval, scheduled, taken
```
will add comment like this